backend/helpers/csv_reader.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework.response import Response
+from rest_framework import status
+from books.models import Book
+from helpers.csv_format_checker import CSVFormatChecker
+from rest_framework.request import Request
+from helpers.csv_exceptions import ExtraHeadersException, MissingHeadersException
+import pandas as pd
+
+
+class CSVReader:
+
+    def __init__(self, csv_import_type: str) -> None:
+        self.csv_import_type = csv_import_type
+
+    def read_csv(self, request: Request):
+        response_data = {}
+        csv_format_checker = CSVFormatChecker(self.csv_import_type)
+        csv = request.FILES["file"]
+        try:
+            csv_df = pd.read_csv(csv, dtype=str, keep_default_na=False)
+        except pd.errors.EmptyDataError:
+            return Response({"errors": "empty_csv"}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Not a huge deal but maybe extract this into the errors file like the rest of the errors? Not sure about how the rest of the codebase looks though